v8go.cc

```diff
@@ -1290,6 +1290,27 @@ void ObjectSet(ValuePtr ptr, const char* key, ValuePtr prop_val) {
   obj->Set(local_ctx, key_val, prop_val->ptr.Get(iso)).Check();
 }
 
+RtnStrings ObjectGetPropertyNames(ValuePtr ptr) {
+  LOCAL_OBJECT(ptr);
+  RtnStrings rtn = {};
+
+  MaybeLocal<Array> maybe_names = obj->GetPropertyNames(local_ctx);
+  Local<Array> names = maybe_names.ToLocalChecked();
```

Comment:
This could fail if the isolate is terminated from another thread, so we need to use ToLocal instead of ToLocalChecked (e.g. https://github.com/rogchap/v8go/pull/177 fixed a lot of usage of ToLocalChecked)